docs/user/kind-example-config.yaml

```diff
@@ -1,23 +1,28 @@
 # this config file contains all config fields with comments
-apiVersion: kind.sigs.k8s.io/v1alpha1
 kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+# 1 control plane node and 3 workers
+nodes:
+# the control plane node config
+- role: control-plane
+  # patch the generated kubeadm config with some extra settings
+  kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1alpha3
```

Comment:
Change to `apiVersion: kubeadm.k8s.io/v1beta1`

Reply:
Thanks. These are problematic because they depend on which k8s version is used (needs to match underlying APIs), will change this 👍

Reply:
done